scripts/mailing_lists/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+"""Create mailing lists for all top-level projects
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.mailing_list.utils import create_list as real_create_list
+from website.models import Node
+from website.notifications.model import NotificationSubscription
+from website.notifications.utils import to_subscription_key
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+EVENT = 'mailing_list_events'
+SUBSCRIPTION_TYPE = 'email_transactional'
+
+def fake(*args, **kwargs):
+    # Override actual method to prevent outgoing calls
+    return
+
+def get_targets():
+    return Node.find()
+
+def migrate(dry_run=True):
+    if dry_run:
+        create_list = fake
+    else:
+        create_list = real_create_list
+
+    successful_enables = []
+    successful_disables = []
+    unknown_failures = {}
+    nodes = get_targets()
+    ncount = len(nodes)
+    logger.info('Preparing to migrate {} nodes.'.format(ncount))
+    for i, node in enumerate(nodes):
+        if node.is_registration or node.is_collection or node.is_deleted:
```

Comment:
I'm a little concerned that if we add node types (registration/project/fork/collection etc) in the future, we'll end up creating mailing lists by accident, because of the need to manually check for a flag in the affirmative. 

Whether in this PR or soon, we should look for ways to make this maintainable, by centralizing the definition of what constitutes a project- scattering references to each new flag throughout the codebase is an invitation to problems down the line.

(I recently worked on a PR involving queries for projects- there were many inconsistent definitions, all scattered throughout the OSF)